### ISSUE TITLE
Modifier keys in mouse.lua on Mac

### ISSUE THREAD

**Issue by DennisSchiefer** [otfried/ipe#46]
I am trying to make use of all modifier keys on Mac (Shift, Control, Option, Command - which should map to Shift - Ctrl - Alt - Meta). So far, however, I am unable to assign the Meta key in any command.

In particular, I am trying to map
`left-meta = "select",`
which does nothing.

I am using Ipe 7.2.2. on an Yosemite MacOS X.

**Comment by otfried** [comment]
The fourth modifier is called "Command" on OSX, and Ipe already uses this internally, but it's not yet mapped.  So you have to change the line:

```
    if modifiers.meta then s = s .. "_meta" end
```

in `tools.lua` to

```
    if modifiers.command then s = s .. "_command" end
```

Then you should be able to use a customization:

```
mouse.left_command = "select"
```

**Comment by mnoellenburg** [comment]
Although this issue should be fixed from Ipe 7.2.3 on as far as I can tell from the release notes, I had trouble getting 
`mouse.left_command = "select"`
to work.

Indeed `tools.lua` defines 
`if modifiers.command then s = s .. "_command" end`
but I also had to modify the second if-statement in function MODEL:mouseButtonAction to be 
`
if button == 1 and not modifiers.alt and not modifiers.control and not modifiers.meta and not modifiers.command then
self:startModeTool(modifiers)
else ...`

I guess this should be included in forthcoming releases.

I'm using Ipe 7.2.7 on Mac OS X 10.11.6.
